server/src/main/java/org/apache/druid/segment/realtime/appenderator/StreamAppenderatorDriver.java

```diff
@@ -332,6 +332,7 @@ public ListenableFuture<SegmentsAndCommitMetadata> registerHandoff(SegmentsAndCo
       }
 
       log.debug("Register handoff of segments: [%s]", waitingSegmentIdList);
+      long handoffStartTime = System.currentTimeMillis();
```

Comment:
Better to clarify the intent of a final variable rather than relying on it being "effectively final"
```suggestion
      final long handoffStartTime = System.currentTimeMillis();
```

Reply:
Done